ansible/roles/riakcs/join_cluster.yml

```diff
@@ -15,7 +15,7 @@
 
 #- debug: var=riak_cluster_status
 
-- set_fact: riak_control_node_name="{{ hostvars[riakcs_control]['riak_nodename'] }}"
+- set_fact: riak_control_node_name="{{ riak_nodename }}"
```

Comment:
Yeah, that definitely sounds right.  I'll revert this change.  Still curious what made this line fail that one time then.